apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -40,7 +40,7 @@
 	"	 wm_test softap [ssid] [password]\n"							\
 	"\n station mode options:\n"										\
 	"	 wm_test sta\n"													\
-	"	 wm_test join [ssid] [security mode] [password]\n"				\
+	"	 wm_test join [ssid] [password] [security mode]\n"				\
```

Comment:
why need swap params sequency?

Reply:
In the old version, all the parameters are mandatory, and if the security_mode is unknown, it will not connect the AP. Now, the security_mode is not necessary, but the password is necessary. So move security_mode to the last one, and if it is unknown, user will not need to input it.